app/src/main/java/com/geeksville/mesh/ui/SettingsFragment.kt

```diff
@@ -501,6 +503,18 @@ class SettingsFragment : ScreenFragment("Settings"), Logging {
         }
     }
 
+    private fun checkBTEnabled(
+        warningReason: String = getString(R.string.requires_bluetooth)
+    ) {
+
+        var btAdapter = BluetoothAdapter.getDefaultAdapter();
+
+        if (!(btAdapter.isEnabled())) {
```

Comment:
Bluetooth state is handled via `bluetoothViewModel.enabled.value`, then `BluetoothAdapter` import can also be removed.

```suggestion
        if (bluetoothViewModel.enabled.value == false) {
```